t/01-pdk/02-log/05-set_serialize_value.t

```diff
@@ -75,15 +75,15 @@ self_ref false value must be nil, a number, string, boolean or a non-self-refere
             local pdk = PDK.new()
 
             pdk.log.set_serialize_value("val1", 1)
-            assert(#ngx.ctx.serialize_values == 3, "== 3 ")
+            assert(#ngx.ctx.serialize_values == 1, "== 1 ")
```

Comment:
It feels wrong that # changes. Can you elaborate why this is okay?

Reply:
@jschmid1,

Yes, I basically removed two default `edits` from `http module` side:
```lua
      { key = "request.headers.authorization", value = "REDACTED", mode = "replace" },
      { key = "request.headers.proxy-authorization", value = "REDACTED", mode = "replace" },
```

And replaced them with direct header manipulation:
```lua
        if request_headers["authorization"] ~= nil then
          request_headers["authorization"] = "REDACTED"
        end
        if request_headers["proxy-authorization"] ~= nil then
          request_headers["proxy-authorization"] = "REDACTED"
        end
```        

This makes it possible to by default not to modify the created table.

Reply:
ok, if we can verify that this doesn't change any user-facing behavior, I'm good with this